hummingbot/client/hummingbot_application.py

```diff
@@ -1139,6 +1140,85 @@ def compare_balance_snapshots(self):
         lines = ["", "  Performance:"] + ["    " + line for line in str(df).split("\n")]
         self._notify("\n".join(lines))
 
+    def add_balances(self, msp, asset_type, asset_time, base, quote):
+        market_name = msp.market.name
+
+        if asset_type == "base":
+            asset_list = base
+            asset = msp.base_asset
+        elif asset_type == "quote":
+            asset_list = quote
+            asset = msp.quote_asset
+        else:
+            raise ValueError("asset_type must be either 'base' or 'quote'")
+
+        if asset_time == "starting":
+            amount = self.starting_balances.get(asset).get(market_name)
+        elif asset_time == "current":
+            amount = self.balance_snapshot().get(asset).get(market_name)
+        else:
+            raise ValueError("asset_time must be either 'starting' or 'current'")
+
+        if len(asset_list) == 0:
+            asset_list.insert(0, asset)
```

Comment:
use list.append instead

Reply:
seems like your list only ever has 2 items. code will be more clear if you use individual variables

Reply:
Replaced list with tuple